src/rpc/rpcquorums.cpp

```diff
@@ -13,6 +13,35 @@
 #include "rpc/server.h"
 #include "validation.h"
 
+UniValue getminedcommitment(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 2) {
+        throw std::runtime_error(
+                "getminedcommitment llmq_type quorum_hash\n"
+                "Return information about the commitment for given quorum.\n"
+                "\nArguments:\n"
+                "1. llmq_type         (number, required) LLMQ type.\n"
+                "2. quorum_hash       (hex string, required) LLMQ hash.\n"
+                "\nExamples:\n"
+                + HelpExampleRpc("getminedcommitment", "2 \"xxx\"")
+                + HelpExampleCli("getminedcommitment", "2, \"xxx\"")
+        );
+    }
+
+    Consensus::LLMQType llmq_type = (Consensus::LLMQType) request.params[0].get_int();
```

Comment:
c30bdc45:
tiny nit: should verify int value before casting it into an `Consensus::LLMQType`.